CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 - [#2931](https://github.com/thanos-io/thanos/pull/2931) Query: Allow passing a `storeMatcher[]` to select matching stores when debugging the querier. See [documentation](https://thanos.io/components/query.md/#store-filtering)
 - [#2991](https://github.com/thanos-io/thanos/pull/2991) store: `operation` label value `getrange` changed to `get_range` for `thanos_store_bucket_cache_operation_requests_total` and `thanos_store_bucket_cache_operation_hits_total` to be consistent with bucket operation metrics.
 - [#2876](https://github.com/thanos-io/thanos/pull/2876) Receive,Ruler: Updated TSDB and switched to ChunkIterators instead of sample one, which avoids unnecessary decoding / encoding.
+- [#3029](https://github.com/thanos-io/thanos/pull/3029) Replace leaktest with goleak
```

Comment:
Let's remove this, it's not user impacting change (: (not implicitly)

Reply:
removed, thanks